conduct.md

```diff
@@ -68,13 +68,13 @@ TODO: Write an introductory paragraph.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Participation for students will be a multidide of factors, we all have bad days, we all have topics we know more or less about, but it is being prepared each day having done the homework, coming to class with an open mind to learn and making an effort to be apart of either small group or large group conversations that will reflect students participation.
```

Comment:
The first sentence seems to be a bit hard to follow for me, I think the bit after `but it` is what's confusing me? Is there a way you can make that flow better? 

The first sentence is a run on as well I think.